presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1522,6 +1523,11 @@ public SystemSessionProperties(
                         "Track history based plan statistics service in query optimizer",
                         featuresConfig.isTrackHistoryBasedPlanStatistics(),
                         false),
+                booleanProperty(
+                        TRACK_HISTORY_STATS_FROM_FAILED_QUERY,
+                        "Track history based plan statistics from complete plan fragments in failed queries",
+                        true,
```

Comment:
this needs a matching configuration property that it should be getting its default values from